### PR TITLE
[VEN-389] Resolve Lease order

### PR DESCRIPTION
**Pull request by a-rmz**
## Description :sparkles:
* Add a `unique` constraint for `leases` associated to `order`s.
* Expose _the_ `order` associated to a `lease` on the model
* Expose _the_ `order` associated to a `lease` on the API
* Add a test for that

### Closes :no_good_woman:
**[VEN-398](https://helsinkisolutionoffice.atlassian.net/browse/VEN-XXX)**

### Automated tests :gear:️
```shell
$ pytest leases/tests/test_lease_queries.py::test_query_berth_lease
```

### Manual testing :construction_worker_man:
**Unique leases**
1. Create an order associated to a lease
2. Try to create another order associated to the same lease, it should raise an error

**Exposed order**
```graphql
query BerthLease {
  berthLease(id: "") {
    id
    order {
      id
    }
  }
}
```

### CODE REVIEW

File: leases/models.py [hunk 1]
```diff
@@ -109,13 +110,22 @@ class BerthLease(AbstractLease):
     renew_automatically = models.BooleanField(
         verbose_name=_("renew automatically"), default=True
     )
+    _orders_relation = GenericRelation(
+        "payments.Order",
+        object_id_field="_lease_object_id",
+        content_type_field="_lease_content_type",
+    )
```
Comment (igordavydsson) [inline leases/models.py:117]: this does not cause new migration files?

Comment (a-rmz) [reply leases/models.py:117]: Nope.

Comment (igordavydsson) [reply leases/models.py:117]: gr8

**Comment by a-rmz** [comment]
:happy-cat:

**Pull request merged**